### PR TITLE
Emailing and password recovery fix

### PR DESCRIPTION
**Pull request by primozs**

### CODE REVIEW

File: skylines/lib/emails.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+from threading import Thread
+from flask import current_app, copy_current_request_context
+from flask_mail import Message
+
+
+def send_async(msg):
```
Comment (TobiasLohner) [inline skylines/lib/emails.py]: Wouldn't it be useful to use our celery backend to send emails?

File: skylines/lib/util.py [hunk 2]
```diff
@@ -1,9 +1,12 @@
+from itsdangerous import Signer, BadSignature
+
+
 def str_to_bool(str):
     return str.lower() in ['1', 'true', 't', 'yes', 'y']
 
 
 def pressure_alt_to_qnh_alt(altitude, qnh):
-    if qnh == None:
+    if qnh is None:
```
Comment (TobiasLohner) [inline skylines/lib/util.py]: Please put bug fixes into a separate pull request.

Comment (Turbo87) [inline skylines/lib/util.py]: IIRC that is already fixed in master now

File: skylines/lib/emails.py (hunk 1 above)
Comment (primozs) [inline skylines/lib/emails.py]: Sure it sounds good, but I would need to look into that, do you have a suggestion how to do it?

File: skylines/lib/util.py (hunk 2 above)
Comment (primozs) [inline skylines/lib/util.py]: Hi, Yes its seams it is. Otherwise my bad to put it here.

Comment (TobiasLohner) [inline skylines/lib/util.py]: Yes, please.

File: skylines/lib/emails.py (hunk 1 above)
Comment (TobiasLohner) [inline skylines/lib/emails.py]: Have a look at https://github.com/skylines-project/skylines/blob/master/skylines/worker/tasks.py#L12 and https://github.com/skylines-project/skylines/blob/master/skylines/frontend/views/upload.py#L418 how to invoke it. It's quite simple.

**Comment by primozs** [comment]
@TobiasLohner Let me know what you think.

**Comment by Turbo87** [comment]
@primozs sorry for not responding on this PR at all. I'm willing to rebase this and fix any conflicts, but can you tell me what this PR was supposed to "fix"? I'm not aware of any issues in that area of the code.

**Comment by Turbo87** [comment]
closing due to inactivity
